### PR TITLE
readWrite chaincode and benchmark fixes

### PR DESCRIPTION
**Pull request by fraVlaca**
- read-write asset tx and benchmark
        - chaincode
            - fixed asset and fixed asset base node
                - deleteAssetFromBatch
                    - delete console.log inside for loop
                    - take out curlies
                - readWriteAsset
                    - update function to new code
            - fixed asset and fixed asset base go
                - readWriteAsset
                    - update function to new code and remove unsafe import
                    - remove brackets
                - deleteasset and delte assetsfrombatch
                    - change uuid to key 
                    - delete log in side the for loop in deleteFromBatch
                    - change delete asset and delte assetfrom bach log 
        - workload
            - change default case for switch , no curly braces beause it is in a single line
            - change from JSON.stringify(letter) to just letter
            - delete comments  
            - change keys for writeOptions to match options of benchamark, hten update switch for new keys
        - fix benchmark previoslyRead variable for random round to writeOptions
        - fix delete preloaded asset benchmark

Signed-off-by: fraVlaca <ocsenarf@outlook.com>

### CODE REVIEW

File: src/fabric/api/fixed-asset/node/lib/fixed-asset.js [hunk 1]
```diff
@@ -290,24 +288,27 @@ class Asset extends Contract {
...
         }
 
-        var fixedAsset;
-        const keysToRead = JSON.parse(readIds);
-        for (const id of keysToRead) {
-            const assetAsBytes = await ctx.stub.getState(id);
+        let fixedAssetBytes;
+        const readIDs = JSON.parse(readIds);
+        for (const id of readIDs) {
+            const assetAsBytes = await stub.getState(id);
```
Comment (davidkel) [inline src/fabric/api/fixed-asset/node/lib/fixed-asset.js:294]:
```suggestion
            const assetAsBytes = await ctx.stub.getState(id);
```

File: src/fabric/api/fixed-asset/node/lib/fixed-asset.js [hunk 2]
```diff
@@ -324,7 +325,7 @@ class Asset extends Contract {
      * @param {String} bookmark - the bookmark from which to start the return
      * @returns {JSON} the results of the paginated query and responseMetadata in a JSON object
      */
-    async paginatedRichQuery(ctx, queryString, pagesize,  bookmark = '') {
+    async paginatedRichQuery(ctx, queryString, pagesize,  bookmark) {
```
Comment (davidkel) [inline src/fabric/api/fixed-asset/node/lib/fixed-asset.js:328]: this shouldn't be changed in this PR

File: src/fabric/api/fixed-asset-base/go/contracts/FixedAssetContract.go [hunk 3]
```diff
@@ -212,45 +210,45 @@ func (contract *FixedAssetContract) PaginatedRangeQuery(ctx utils.Context, start
...
+func (contract *FixedAssetContract) ReadWriteAssets(ctx utils.Context, readIds []string, writeIds []string, letter string) error {
+	fmt.Println("Entering ReadWriteAssets")
 
-	var fixedAsset *assets.FixedAsset
+	fixedAsset := assets.FixedAsset{}
+	var bytes []byte
 
-	for _, uuid := range readUuids {
-		bytes, err := ctx.GetStub().GetState(uuid)
+	for _, uuid := range readIds {
```
Comment (davidkel) [inline src/fabric/api/fixed-asset-base/go/contracts/FixedAssetContract.go:220]: change uuid to id here so it's consistent with the rest of the method

File: src/fabric/api/fixed-asset/go/FixedAssetContract.go [hunk 4]
```diff
@@ -295,44 +293,45 @@ func (s *SmartContract) PaginatedRangeQuery(ctx contractapi.TransactionContextIn
...
+func (s *SmartContract) ReadWriteAssets(ctx contractapi.TransactionContextInterface, readIds []string, writeIds []string, letter string) error {
+	fmt.Println("Entering ReadWriteAssets")
 
-	var fixedAsset *assets.FixedAsset
-	for _, uuid := range readUuids {
-		bytes, err := ctx.GetStub().GetState(uuid)
+	fixedAsset := assets.FixedAsset{}
+	var bytes []byte
+
+	for _, key := range readIds {
```
Comment (davidkel) [inline src/fabric/api/fixed-asset/go/FixedAssetContract.go:303]: use id instead of key here to be consistent with the other part of the method

**Review by davidkel — CHANGES_REQUESTED** [review]
Small number of changes

File: src/fabric/api/fixed-asset/node/lib/fixed-asset.js (hunk 2 above)
Comment (fraVlaca) [reply src/fabric/api/fixed-asset/node/lib/fixed-asset.js:328]: ah sorry forgot to take it out my bad

**Review by davidkel — APPROVED** [review]
LGTM

**Pull request merged**